### PR TITLE
Add missing docs for HttpClient.PatchAsync

### LINKED ISSUE #1172: Missing .net core docs
This method is undocumented.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8ce8e507-db84-c2cf-d8e2-56e2c11a50d8
* Version Independent ID: 3489b98b-3252-2f41-89b5-c82a15cd5967
* Content: [HttpClient.PatchAsync Method (System.Net.Http)](https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.patchasync?view=netcore-2.1)
* Content Source: [xml/System.Net.Http/HttpClient.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Net.Http/HttpClient.xml)
* Product: **dotnet-api**
* GitHub Login: @karelz
* Microsoft Alias: **ncldev**

### PR DESCRIPTION
**Pull request by dsychin**
Fixes #1172

Added missing documentation.
Also added missing VB member signature.

### CODE REVIEW

File: xml/System.Net.Http/HttpClient.xml [hunk 1]
```diff
@@ -1462,18 +1477,26 @@ public class GoodController : ApiController
...
-        <param name="requestUri">To be added.</param>
-        <param name="content">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="requestUri">The Uri the request is sent to.</param>
+        <param name="content">The HTTP request content sent to the server.</param>
+        <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
+        <summary>Send a PATCH request with a cancellation token as an asynchronous operation.</summary>
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]:
```suggestion
        <summary>Sends a PATCH request with a cancellation token to a Uri represented as a string as an asynchronous operation.</summary>
```

File: xml/System.Net.Http/HttpClient.xml [hunk 2]
```diff
@@ -1494,12 +1517,19 @@ public class GoodController : ApiController
...
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]:
```suggestion
        <summary>Sends a PATCH request with a cancellation token as an asynchronous operation.</summary>
```

**Comment by mairaw** [comment]
The missing VB.NET signature fix might disappear next time we run reflection. 
We have a tracking bug for this on the mdoc tool:
https://github.com/mono/api-doc-tools/issues/356

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @dsychin. I'll merge your PR now. The new documentation should be live on docs.microsoft.com in the next day or two.

**Pull request merged**